client/src/styles/VerifyEmailPage.scss

```
@@ -0,0 +1,5 @@
+.VerifyemailPage {
```

```suggestion
.VerifyEmailPage {
```
pedantic, but camel case for consistency